.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -5,6 +5,10 @@ set -e
 function clone_repo() {
```

Comment:
Could you test whether the containers behave as expected, at least for generate-diffs?

Reply:
I did test the generate-diff before: https://github.com/GoogleCloudPlatform/magic-modules/pull/7882
But I will test the change in this PR again to make sure it work as expected